models/classification/_base_/densenet_121_omz.py

```diff
@@ -12,7 +12,7 @@
     neck=dict(
         type='GlobalAveragePooling'),
     head=dict(
-        type='LinearClsHead',
+        type='CustomLinearClsHead',
```

Comment:
I think it's better to only change the target models (mv3, effnet).

Reply:
Jihwan says it would be problem in omz model. I will revert and remain [non_linear_cls_head.py](https://github.com/openvinotoolkit/model_preparation_algorithm/pull/64/files/daa790b4b61074c5a4ee115cb1c3faf6d7a27e3f#diff-cea1efa8c06cc9c65e5f674a99dbfff87cd744951fb9666ea67507c8e78427a0)